gix-transport/src/client/blocking_io/http/reqwest/remote.rs

```diff
@@ -3,14 +3,12 @@ use std::{
     convert::TryFrom,
     io::{Read, Write},
     str::FromStr,
+    sync::{atomic, Arc},
 };
 
 use gix_features::io::pipe;
 
-use crate::client::{
-    http,
-    http::{reqwest::Remote, traits::PostBodyDataKind},
-};
+use crate::client::http::{self, options::FollowRedirects, redirect, reqwest::Remote, traits::PostBodyDataKind};
 
 /// The error returned by the 'remote' helper, a purely internal construct to perform http requests.
 #[derive(Debug, thiserror::Error)]
```

Comment:
P.S. I think it will be better if `#[non_exhaustive]` is added to all error types so that adding new variant won't be a breaking change.